packages/fluentui/react-northstar/src/themes/teams/components/Checkbox/checkboxStyles.ts

```diff
@@ -115,9 +117,15 @@ export const checkboxStyles: ComponentSlotStylesPrepared<CheckboxStylesProps, Ch
     backgroundPosition: 'center',
     backgroundRepeat: 'no-repeat',
 
-    ...(p.checked && {
+    ...(p.checked &&
+      p.checked !== 'mixed' && {
```

Comment:
```suggestion
      p.checked && {
```

A next condition will win anyway when `checked` is mixed